presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -185,6 +186,28 @@ public int getQueuedQueries()
         }
     }
 
+    @Managed
+    public int getBlockedQueuedQueries()
```

Comment:
I see why you would want to call this blocked, but we already use the term blocked in a different context within Presto. Maybe 'waiting' is slightly more descriptive?